src/components/Heading.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const StyledHeading = styled.div`
+  color: var(--tints-blizzard-black);
+  text-align: center;
+
+  .brow {
```

Comment:
Looks great! The only difference I caught with the Figma design file was the eyebrowtext. We need to make sure it's all uppercase with CSS.

Reply:
Great catch. Thank you!